src/components/organisms/Table/stories/v2.stories.tsx

```diff
@@ -0,0 +1,306 @@
+  "january": 1463,
+  "february": 616,
+  "march": 1722,
+  "april": 987,
+  "may": 1925,
+  "june": 65,
+  "july": 315,
+  "august": 1777,
+  "september": 1497,
+  "october": 1610,
+  "november": 1248,
+  "december": 866,
+  "average": 1000
+  }
+  ]
+
+const metaData = [
+    {
+    "id": 1,
+    "employee": "forecast",
+    "january": 1519,
+    "february": 390,
+    "march": 1126,
+    "april": 746,
+    "may": 297,
+    "june": 789,
+    "july": 721,
+    "august": 369,
+    "september": 232,
+    "october": 434,
+    "november": 489,
+    "december": 1,
+    "average": 1000
+    },
+    {
+    "id": 2,
+    "employee": "turnover",
+    "january": 1212,
+    "february": 1133,
+    "march": 1116,
+    "april": 1530,
+    "may": 1659,
+    "june": 902,
+    "july": 1321,
+    "august": 1929,
+    "september": 297,
+    "october": 1192,
+    "november": 1957,
+    "december": 1646,
+    "average": 1000
+    },
+    {
+    "id": 3,
+    "employee": "cost",
+    "january": 1057,
+    "february": 268,
+    "march": 1854,
+    "april": 234,
+    "may": 1893,
+    "june": 1311,
+    "july": 532,
+    "august": 171,
+    "september": 871,
+    "october": 627,
+    "november": 592,
+    "december": 928,
+    "average": 1000
+    },
+    {
+    "id": 4,
+    "employee": "difference",
+    "january": 1692,
+    "february": 210,
+    "march": 391,
+    "april": 944,
+    "may": 1641,
+    "june": 809,
+    "july": 202,
+    "august": 270,
+    "september": 1020,
+    "october": 180,
+    "november": 976,
+    "december": 1799,
+    "average": 1000
+    }
+]
+
+export const TableV2: React.FC = () => {
+  const [currentData, setCurrentData] = React.useState<TableRow[]>(userData as TableRow[])
+
+  return (
+    <Table sortData={currentData} originalData={userData} setSortData={setCurrentData}>
+      <Head>
+        <Row>
+          <Cell type='head' value='full name' field='employee' sortable color='success' />
+          <Cell type='head' value='january' field='january' />
+          <Cell type='head' value='february' field='february' />
+          <Cell type='head' value='march' field='march' />
+          <Cell type='head' value='april' field='april' />
+          <Cell type='head' value='may' field='may' />
+          <Cell type='head' value='june' field='june' />
+          <Cell type='head' value='july' field='july' />
+          <Cell type='head' value='august' field='august' />
+          <Cell type='head' value='september' field='september' />
+          <Cell type='head' value='october' field='october' />
+          <Cell type='head' value='november' field='november' />
+          <Cell type='head' value='december' field='december' />
+          <Cell type='head' value='average (2020)' field='average' color='warning' />
+        </Row>
+      </Head>
+      <Body>
+        {currentData.map((row: TableRow) => {
+          return (
+            <Row key={row.id.toString()}>
+              <Cell value={row['employee']} color='success' />
```

Comment:
I guess it makes more sense to use a prop `type` instead of `color` (IMHO as a developer I will think that it impacts the font's color).

Reply:
yeah, we can also do that. i'll update it.